pkg/controller/dnsendpoint/nameserver/test_helpers.go

```diff
@@ -0,0 +1,11 @@
+package nameserver
```

Comment:
Rename this file to `helpers_test.go` so it is only compiled with tests.

Reply:
Done